### DISCUSSION TITLE
Advanced Settings and Delete

### DISCUSSION THREAD

**Discussion by simmeee** [photoprism/photoprism#2748] [General]
I installed the latest version (220919-cc8bab446) and I noticed two things.
When I go to Settings - Advanced, I am always redirected to the Search page.
When I select images under Archives, I am missing the delete button.
Can anyone confirm this or does it only affect my installation?

**Comment by lastzero** [comment]
Seems to be our development preview image?These are the kind of issues you would expect while we are working on multi user functionality.

**Reply by simmeee** [reply]
I was importing images from my iPhone and had the rotation issue, so I switched to the Developer version. I'll wait for the next stable version then. Thanks for the feedback.
